docs/functions/util.rst

```diff
@@ -143,40 +180,28 @@ These functions are designed to be used by themes.
 
    This function will do nothing if :attr:`LP_ENABLE_TITLE` is disabled.
 
-.. function:: _lp_formatted_title(title)
-
-   Sets the theme generated title to *title*. The input is escaped using
-   :func:`__lp_strip_escapes` to strip terminal formatting from being added to
-   the title.
-
-   This function will do nothing and return ``2`` if :attr:`LP_ENABLE_TITLE`
-   is disabled.
-
-   .. versionadded:: 2.0
-
-.. function:: _lp_raw_title(title)
+.. function:: _lp_version_greatereq([major, [minor, [patch, [string, [number]]]]])
```

Comment:
This signature does not match the inline docs and code. Major and minor are not optional.

```suggestion
.. function:: _lp_version_string(major, minor, [patch, [string, [number]]]) -> var:lp_version
```

Reply:
Your diff is ambiguous. I corrected `_lp_version_greatereq`, but for `_lp_version_string`, all parametert are optional (to allow for collecting the current version string).